>From eaeb55be19ca37c7a09d04bc9d6160cf9d03a145 Mon Sep 17 00:00:00 2001
From: Daniel Kahn Gillmor <dkg@fifthhorseman.net>
Date: Thu, 30 Jun 2016 14:02:16 -0400
Subject: [PATCH 2/2] handle dkms conf files where PACKAGE_VERSION or
 PACKAGE_NAME is unquoted

Some dkms.conf files do not quote PACKAGE_VERSION or PACKAGE_NAME,
like:

PACKAGE_NAME=blah
PACKAGE_VERSION=0.0.0

This patch lets dh_dkms grok this syntax.
---
 debian/scripts/dh_dkms | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/debian/scripts/dh_dkms b/debian/scripts/dh_dkms
index f5a1c4b..48fe8b3 100755
--- a/debian/scripts/dh_dkms
+++ b/debian/scripts/dh_dkms
@@ -116,8 +116,8 @@ foreach my $package (@{$dh{DOPACKAGES}}) {
 	# now, parse our configuration file
 	open(IN, "< $name");
 	while (my $l = <IN>) {
-		$l =~ /PACKAGE_NAME=(["'])(.*)\1/ && ($is_snippet = 1 && $package_name = $2);
-		$l =~ /PACKAGE_VERSION=(["'])(.*)\1/ && ($package_version = $2);
+		$l =~ /PACKAGE_NAME=(["']?)(.*)\1/ && ($is_snippet = 1 && $package_name = $2);
+		$l =~ /PACKAGE_VERSION=(["']?)(.*)\1/ && ($package_version = $2);
 	}
 	close(IN);
 
-- 
2.8.1

